en/publications.md

```diff
@@ -7,8 +7,24 @@ permalink: /en/publications/
 share: false
 version: 1
 ---
+{%- if jekyll.environment == 'future' -%}
+- [Scaling book][]: a guide to effective practices and techniques
+  that engineers interacting with the Bitcoin block chain can adopt
+  today.
+
+- [Blog posts][]: Occasional updates and reference material
+  from the Optech Team.
+
+- [Newsletters][]: a weekly summary of news about Bitcion and LN
+  development.
+
+<br><br>A sample of recent publications from our [blog posts][] and [newsletters][]:
```

Comment:
These anchors for 'blog posts' and 'newsletters' seem redundant with the list above. Perhaps just change this to "Recent publications:"

Reply:
fixed